nilrt_snac/__main__.py

```diff
@@ -3,18 +3,17 @@
 import argparse
 import logging
 import sys
-from typing import List, Optional
+import configparser
```

Comment:
Some of your changes in this PR deal with formatting (quotes, etc.). So my mind immediately goes to tools like black. Are you making use of that? If not, should we do so for this tool?

The reason I ask is because I know multiple imports are usually alpha sorted, and if they're in different logical groups (I use that term though I can't define it) then they're separated by a blank line - but this is neither.